data/common.yaml

```diff
@@ -1 +1,22 @@
---- {}
+---
+lookup_options:
+  profile_java::pkg_list:
+    merge:
+      strategy: "deep"
+      knockout_prefix: "-"
+
+profile_java::install_lock_files: true
+profile_java::lock_files:
+  "/etc":
+    ensure: "directory"
+  "/etc/.java":
+    ensure: "directory"
+  "/etc/.java/.systemPrefs":
+    ensure: "directory"
+  "/etc/.java/.systemPrefs/.system.lock":
+    ensure: "file"
+  "/etc/.java/.systemPrefs/.systemRootModFile":
+    ensure: "file"
+profile_java::pkg_list:
+  "RedHat":
+    "java-1.8.0-openjdk-headless":
```

Comment:
Generally within a shared profile  (or shared module) like this, I would create OS specific directories under the `data` directory, rather than specifying the OS in the hash. I think it is a bit more obvious for the standard user that uses the module. To my knowledge `profile_additional_pkgs` is the only one that specifies OS version in the hash.

Reply:
Yep, in fact, that's where I borrowed the code from (profile_additional_pkgs). I'll update to use the more standard pattern.

I raised the version question w/ the System Management team an we decided to stick w/ 1.8.0 (essentially the default from Red Hat). We expect that on clusters anyway, newer versions will generally be installed in a shared filesystem, if needed.

Reply:
Done!